### PR TITLE
Perftest fix

### PR DESCRIPTION
**Pull request by Thor-tech-of-metal**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: charts/ccd-definition-store-api/values.aat.template.yaml [hunk 1]
```diff
@@ -2,4 +2,14 @@ java:
   # Don't modify below here
```
Comment (mario-paniccia) [inline charts/ccd-definition-store-api/values.aat.template.yaml]: this is not the correct file. This is the AAT file. You have to change `values.yaml`

File: charts/ccd-definition-store-api/values.aat.template.yaml [hunk 2]
```diff
@@ -2,4 +2,14 @@ java:
...
+    DEFINITION_STORE_DB_HOST: "{{ .Release.Name }}-postgresql"
+    DEFINITION_STORE_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    DEFINITION_STORE_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    DEFINITION_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    DEFINITION_STORE_DB_OPTIONS: "?sslmode=require"
+    DEFINITION_STORE_DB_MAX_POOL_SIZE: 5
 
+    ENABLE_DB_MIGRATE: true
+    postgresql:
+      enabled: true
```
Comment (mario-paniccia) [inline charts/ccd-definition-store-api/values.aat.template.yaml]: this is not correct, why we are switching from azure to aks db?

File: charts/ccd-definition-store-api/values.aat.template.yaml [hunk 3]
```diff
@@ -2,4 +2,14 @@ java:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
+    DEFINITION_STORE_DB_HOST: "{{ .Release.Name }}-postgresql"
+    DEFINITION_STORE_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
+    DEFINITION_STORE_DB_USERNAME: "{{ .Values.postgresql.postgresqlUsername}}"
+    DEFINITION_STORE_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    DEFINITION_STORE_DB_OPTIONS: "?sslmode=require"
```
Comment (mario-paniccia) [inline charts/ccd-definition-store-api/values.aat.template.yaml]: this is not correct, why we are switching from azure to aks db?

Comment (Thor-tech-of-metal) [reply charts/ccd-definition-store-api/values.aat.template.yaml]: I agree  the host to be used is the one in values .  ccd-definition-store-api-postgres-db-{{ .Values.global.environment }}.postgres.database.azure.com .

File: charts/ccd-definition-store-api/values.aat.template.yaml (hunk 1 above)
Comment (Thor-tech-of-metal) [reply charts/ccd-definition-store-api/values.aat.template.yaml]: The changes were supposed to  done for performance test only . But Do we want perfTest env to be exactly the same as the default  values.yml?  If it is the case I am only going to change the pool connection in the value.yml to the last known configuration 16  from /infrastructure

**Pull request merged**